.github/workflows/clang-tidy.yml

```diff
@@ -13,9 +13,11 @@ jobs:
     runs-on: ubuntu-latest
     container: px4io/px4-dev-clang:2021-09-08
     steps:
-    - uses: actions/checkout@v1
+    - uses: actions/checkout@v2
```

Comment:
bump to @v4

Reply:
We can't do the old container. Even v3 is too new.